### DISCUSSION TITLE
v4.32升级v4.37把config.json配置文件错误覆盖

### DISCUSSION THREAD

**Discussion by wuwentao** [v2fly/v2ray-core#923] [General]
从v4.32升级v4.37，ubuntu 20.04，直接unzip压缩包覆盖提示选All了，才反应过来config.json被彻底覆盖了
服务器当前没重启，还运行着v4.32的进程，内存中应该有当前配置文件的参数，请问是否有办法把当前配置文件dump出来或者参数读取出来，要不挨个挨个恢复就比较麻烦了！

**Comment by Cwek** [comment]
或者试试Google下 go memory dumping ？
